doc/user/tutorial.rst

```diff
@@ -559,11 +559,8 @@ entire resource like so:
         # ...
 
 And you can apply hooks globally by passing them into the API class
-initializer:
-
-.. code:: python
-
-    falcon.API(before=[extract_project_id])
+initializer, though this has been depreciated in favor of middleware.
```

Comment:
I think it may be even better to simply not mention how hooks can be applied globally, specifically:
 * In L551, remove the "or even globally" clause.
 * In L561-L563, say something about how logic can also be applied globally using middleware (with no mention of global hooks)

Typo: "depreciated" ==> "deprecated"